src/my-work-provider.ts

```diff
@@ -55,6 +80,7 @@ export abstract class MyWorkProvider implements vscode.TreeDataProvider<MyWorkIt
 export class MyWorkItem extends vscode.TreeItem {
 
     public entity?: OctaneEntity;
+    public iconPath?: vscode.Uri;
```

Comment:
Please check if `iconPath` is an existing attribute of `TreeItem`. If it is, we shouldn't duplicate the attribute.

Reply:
Yes, it is an attribute of `TreeItem`. Fixed issue.